lib/iris/cube.py

```diff
@@ -1050,6 +1114,25 @@ def coord_dims(self, coord):
                        factories]
 
         if not matches:
+            raise iris.exceptions.CellMeasureNotFoundError(coord.name())
```

Comment:
@marqh Oops! Something has gone wrong here, perhaps due to some rebasing issues?

Change this back to ...

```python
if not matches:
    raise iris.exceptions.CoordinateNotFoundError(coord.name())
```

Interesting how this change didn't cause any tests to fail ...